sormas-api/src/main/java/de/symeda/sormas/api/exposure/ExposureDto.java

```diff
@@ -242,6 +243,7 @@
 	private String deceasedPersonName;
 	@SensitiveData
 	private String deceasedPersonRelation;
+	private YesNoUnknown largeAttendenceNumber;
```

Comment:
=> largeAttendanceNumber (has to be changed everywhere)

Reply:
I just used the term from the issue. It then isn't correct there too.

Reply:
That's right - sorry, my bad!